src/WatchDogs.Application/Program.cs

```diff
@@ -97,7 +109,9 @@
 
         var loadedTrades = await SDD.LoadDealsAsync();
 
-        await SDD.DetectSuspiciousDealsAsync(loadedTrades);
+        var SS = await SDD.DetectSuspiciousDealsAsync(loadedTrades);
```

Comment:
Why is the using scope still there? I thought we would move it all to `FakeWatcher`

Reply:
testing purposes